README.md

```diff
@@ -95,6 +95,24 @@ There is a [full feature example of use](https://github.com/willgm/web-crypto-st
 
 This project is heavily inspired on [Tim Taubert](https://twitter.com/ttaubert)'s talk at JSConf EU: [Keeping secrets with JavaScript](https://www.youtube.com/watch?v=yf4m9LdO1zI), if you want to better understand what happens under the hood, then I highly recommend for you to watch it out.
 
+### Crypto Key FAQ
+
+#### What should I use as the crypto key?
+
+There is no perfect answer for this question, it really depends on your objective using this library and the application itself. If you do want to have a strong security, and make as difficult as possible to someone not authorized to access the data, then the crypto key should be unique by user, something strong that only the he knows and it is never stored anywhere out of his mind, as the **user password**. But it is not the only possible strategy, you can do any sort of server integration to obtain a crypto key, by session or any other thing that makes sense to your product logic or application platform.
```

Comment:
There is no perfect answer to this question, it depends on your objective using this library and the application itself. If you do want to have strong security and make as difficult as possible to someone not authorized to access the data, then the crypto key should be unique by a user, something strong that only the user knows and it is never stored anywhere out of his mind, as the user password. But it is not the only one possible strategy, you can create a server integration to obtain a crypto key, by session, or any other thing that makes sense to your product logic or application platform.

Reply:
I went all the way through your paragraph to find the changes and adopted most of it, but I still kept some as they are or changed in a different way. Thanks!